src/_guides/language/language-tour.md

```diff
@@ -758,7 +758,7 @@ units. Use the `runes` property to get the runes of a string.
 
 The following example illustrates the relationship between runes,
 16-bit code units, and 32-bit code points.
-Click the run button ( {% asset red-run.png %} )
+Click the run button {% asset red-run.png alt="" %}
```

Comment:
Maybe `alt="(>)"`, here and elsewhere?

Reply:
(Btw, my plan was to eventually replace these images with the appropriate Material Icon.)

Reply:
The image doesn't add anything for non-sighted users, so per the accessibility guidelines I've found (e.g. https://webaim.org/techniques/alttext/), I set it to an empty string.

Reply:
(These are screenshots, so I'm not sure the Material icon is appropriate here.)

Reply:
We don't actually want a screenshot here, we want to show a button (which we can color red if that is deemed necessary):

    Click the run button <i class="material-icons">play_circle_filled</i>.

As you can see from this DartPad screenshot:

> <img width="351" alt="screen shot 2018-08-31 at 05 57 41" src="https://user-images.githubusercontent.com/4140793/44906418-d7ecb200-ace2-11e8-8403-41cff639ec30.png">

The DartPad run button is essentially a standard Material play button, so it seems appropriate IMHO:

> <img width="304" alt="screen shot 2018-08-31 at 06 00 04" src="https://user-images.githubusercontent.com/4140793/44906537-3023b400-ace3-11e8-9ac8-59753efdbaf5.png">

Let me know if you agree. If so, I'll create a followup issue.

Reply:
Sure, that sounds good to me. I didn't realize the DartPad button conformed to Material Design. :)

Reply:
Done: #1118